docs/src/security.md

```diff
@@ -367,3 +388,66 @@ For further detail on how `pg_ident.conf` is managed by the operator, see the
 CloudNativePG delegates encryption at rest to the underlying storage class. For
 data protection in production environments, we highly recommend that you choose
 a storage class that supports encryption at rest.
+
+## Cluster wide permissions and operator deployment
+
+### Using manifest
+
+In the default installation for a vanilla Kubernetes cluster, the operator is deployed
+using the manifest that is build using `Kustomize` on the yaml files generated
+by `controller-gen` using the decorations available on [Kubebuilder](https://book.kubebuilder.io/),
```

Comment:
"decorations" ...???  Do you mean declarations? Customizations? Or is declaration a part of the lingo, like Ansible "playbooks"?